force-app/main/SDO_SCOM_TNC_ACKNOWLEDGEMENT/lwc/sdo_scom_tnc_acknowledgement_place_order/sdo_scom_tnc_acknowledgement_place_order.html

```diff
@@ -20,10 +20,4 @@
             {error}
```

Comment:
You can use `setCustomValidity` on lighitng-input` instead of using our own html, so that it will cover accessibility also.